app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -175,10 +183,53 @@ private void updateObstaclesCoordinates() {
             if (obstacle.getX() + obstacle.getWidth() < 0) {
                 it.remove();
                 score += 10;
+                if (activity.getSavedSignedInAccount() != null) {
```

Comment:
As a potential upgrade: I think we should also store these scores when user is not signed it, and merge these scores with scores from the player profile when they sign in eventually. Generally, player progress should not be cleared when user played not signed in for some time and then sign in afterwards.

Reply:
But what should we do in case user log out and sign in with another account?